src/client/opamClient.ml

```diff
@@ -850,15 +850,15 @@ let windows_checks ?cygwin_setup ?git_location config =
            OpamFilename.(Dir.to_string (dirname_dir (dirname_dir cygbin))));
     config
   in
-  let install_cygwin_tools () =
-    let packages =
+  let install_cygwin_tools packages =
+    let default_packages =
       match OpamSystem.resolve_command "git" with
       | None -> OpamInitDefaults.required_packages_for_cygwin
       | Some _ ->
         List.filter (fun c -> not OpamSysPkg.(equal (of_string "git") c))
           OpamInitDefaults.required_packages_for_cygwin
     in
-    OpamSysInteract.Cygwin.install ~packages
+    OpamSysInteract.Cygwin.install ~packages:(default_packages @ packages)
```

Comment:
It's worth a comment here that `packages` comes last, because it allows the command line to override version constraints in the default (were we to put any)